modules/programs/git-worktree-switcher.nix

```diff
@@ -0,0 +1,52 @@
+{ pkgs, config, lib, ... }:
+
+let
+  inherit (lib) mkEnableOption mkOption mkPackageOption optionalString;
+
+  cfg = config.programs.git-worktree-switcher;
+
+  initScript = shell:
+    if (shell == "fish") then ''
+      ${lib.getExe pkgs.git-worktree-switcher} init ${shell} | source
+    '' else ''
+      eval "$(${lib.getExe pkgs.git-worktree-switcher} init ${shell})"
+    '';
+in {
+  meta.maintainers = [ lib.maintainers.mateusauler ];
```

Comment:
The home-manager maintainer you set up isn't even used here. See how a `home-manager` maintainer is used elsewhere:

https://github.com/nix-community/home-manager/blob/8264bfe3a064d704c57df91e34b795b6ac7bad9e/modules/programs/thefuck.nix#L6